docs/en/operations/update.md

```diff
@@ -16,3 +16,15 @@ $ sudo service clickhouse-server restart
 If you installed ClickHouse using something other than the recommended `deb` packages, use the appropriate update method.
 
 ClickHouse does not support a distributed update. The operation should be performed consecutively on each separate server. Do not update all the servers on a cluster simultaneously, or the cluster will be unavailable for some time.
+
+The upgrade of older version of ClickHouse to specific version:
+
+As an example: 
+
+```bash
+$ sudo apt-get update
+$ sudo apt-get install clickhouse-server=20.12.4.5 clickhouse-client=20.12.4.5 clickhouse-common-static=20.12.4.5
+$ sudo service clickhouse-server restart
+```
+
+Note: It's always recommended to backup all databases before initiating the upgrade process. Please make sure the new version is compatible with new changes so on.
```

Comment:
In fact we guarantee that all data formats are compatible.
Backup before upgrade is often impractical on large clusters.

When you install patch release (e.g. 20.8.11 -> 20.8.12), it's recommended to install is ASAP without any concerns.

Reply:
Okay! Thanks for this information.  I can remove this note